lua/notes.lua

```diff
@@ -3,6 +3,13 @@ local Note = require('notes.note')
 
 local M = {}
 
+-- Helper function to calculate previous day's date
+local function get_previous_day_date()
```

Comment:
Instead of making it local and then exporting the function for testing, just make it public.